mne/utils.py

```diff
@@ -2744,13 +2737,16 @@ def open_docs(kind=None, version=None):
 
 def _validate_type(item, types, item_name, type_name=None):
     """Validate that `item` is an instance of `types`."""
-    if type_name is None:
-        if types == string_types:
-            type_name = "str"
-        else:
-            iter_types = ([types] if not isinstance(types, (list, tuple))
-                          else types)
-            type_name = ', '.join(cls.__name__ for cls in iter_types)
-    if not isinstance(item, types):
-        raise TypeError(item_name, ' must be an instance of ', type_name,
-                        ', got %s instead.' % (type(item),))
+    if types == Integral:
```

Comment:
then this can probably be `if types is int`

Reply:
Hm, maybe the best solution is to allow not only types, but strings there - in particular

'str', 'int', 'numeric', 'BaseEpochs', 'BaseRaw', 'Evoked'.

Cause these come up often.

The 'numeric' option would allow removing the `_is_numeric` function I wrote for a viz method ...

Reply:
Sure, passing strings for common cases sounds good. Here I think the minimum we need is `types == 'int'`, but feel free to add the others if you want, too.